src/app/app.component.ts

```diff
@@ -1,10 +1,72 @@
-import { Component } from '@angular/core';
+import { Component, OnInit, ViewChild } from '@angular/core';
+
+import { AppConstants } from './shared/constants';
+import { FileService } from './shared/util';
 
 @Component({
-  selector: 'app-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+    selector: 'app-root',
+    templateUrl: './app.component.html',
+    styleUrls: ['./app.component.css'],
+    providers: [
+        AppConstants,
+        FileService
+    ]
 })
-export class AppComponent {
-  title = 'app';
+export class AppComponent implements OnInit {
+    title = 'app';
+
+    @ViewChild('fileImportInput')
+    fileImportInput: any;
+
+    csvRecords = [];
+
+    constructor(
+        private _fileUtil: FileService
```

Comment:
Ideally we should have used a file uploader component to do this and not app component